books/free-programming-books.md

```diff
@@ -2292,6 +2292,7 @@ Kerridge (PDF) (email address *requested*, not required)
 
 * [Postgres Official Documentation](http://www.postgresql.org/docs/)
 * [Postgres Succinctly](https://www.syncfusion.com/resources/techportal/ebooks/postgres) (PDF, Kindle) (email address *requested*, not required)
+* [PostgreSQL Tutorial](https://www.tutorialspoint.com/postgresql) - Tutorials Point
```

Comment:
OK, I see that the books list doesn't group the various dialects of SQL together, as some of our lists do. I guess that should be a separate PR. I will create an issue.